src/dune_rules/melange/melange_rules.ml

```diff
@@ -275,13 +333,19 @@ let setup_entries_js ~sctx ~dir ~dir_contents ~scope ~compile_info ~target_dir
   in
   let output = `Private_library_or_emit target_dir in
   let obj_dir = Obj_dir.of_local obj_dir in
+  let* (_runtime_deps : unit Action_builder.t) =
```

Comment:
I don't understand why `setup_runtime_assets_rules` returns something different than `unit` if it is not used.

Reply:
They should just be attached to the alias. Otherwise, it would slow down the build.